src/calculator.js

```diff
@@ -36,14 +36,14 @@ exports.multiply = (x, y) => {
   return x * y;
 };
 
-exports.divide = (x, y) => {
+exports.divide = (x,y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
```

Comment:
Missing semicolon.

###### See http://eslint.org/docs/rules/semi for details about this rule